src/components/ContactField/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { memo } from 'react';
+import { Text, View, Pressable, TextInputProps } from 'react-native';
+
+import { Input, Label } from '../Input';
+import styles from './styles';
+
+type ContactFieldProps = TextInputProps & {
+  label: string;
+  error?: string;
+  style?: object;
+  isEditing?: boolean;
+  onPress?: () => void;
+};
+
+const ContactField = (props: ContactFieldProps) => {
+  const {
+    label,
+    value,
+    error,
+    style = {},
+    isEditing = false,
+    onPress,
+    ...restOfProps
+  } = props;
+  return (
+    <View style={[style, styles.textInputContainer]}>
+      <Label labelStyle={styles.label} label={label} />
+      {onPress && <Pressable style={styles.pressableArea} onPress={onPress} />}
```

Comment:
wrap entire component with Pressable, using absolute style here may create issue

Reply:
Hello, I tested. It's does not create any issues. This pressable case is just a special case, therefore, I don't want to wrap the entire component with Pressable. This type wrap can effect the input's press.

Reply:
Pressable has "disabled" props, you can use it to deactivate it, I don't know your use case

Reply:
yes, my case is different. If I set disabled I can not touch text input. this uses for the same UI without duplicate code.

Reply:
yes, my case is different. If I set disabled I can not touch text input. this uses for the same UI without duplicate code.

Reply:
Which component will be active if I will send onPress and isEditing=true?

Reply:
its for open the date-picker component. I don't want to take text, but I want to display the same component.

Reply:
did you try to use onPressIn of TextInput?

Reply:
yes, but it's not working only when the user press the component. It also works when a user tries to scroll the screen.

Reply:
by the way, onPressOut is the same